imports/fixture/fixture-builder.js

```diff
@@ -48,12 +48,14 @@ export class FixtureBuilder {
   }
   createParcel(doc) {
     const ref = 'A' + doc.floor + doc.door;
+    const volume = 3 * doc.area || 0;
```

Comment:
Nem azt akartad irni, hogy 3 * (doc.area || 0)?
Vajon a 3 * undefined, az mennyit ad javascriptben.

Reply:
3 * undefined = NaN, így a NaN || 0 képletből a 0-t kapjuk vissza. Jó ez így is. :)